simulator/obj_canvas.py

```diff
@@ -404,3 +404,18 @@ def _draw_object(self):
   def get_bbox(self):
     # TODO (danielp): Real bounding box calculation.
     return (self._pos_x, self._pos_y, self._pos_x, self._pos_y)
+
+	
+class Grid(CanvasObject):
```

Comment:
Does this actually get used anywhere? If not, let's remove it and save it for another PR.